src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -37,10 +41,14 @@ export class MdTabNavBar implements AfterContentInit, OnDestroy {
   /** Combines listeners that will re-align the ink bar whenever they're invoked. */
   private _realignInkBar: Subscription = null;
 
+  /** Subject that emits when the component has been destroyed. */
+  private _onDestroy = new Subject<void>();
```

Comment:
Both work, playing with the idea of using this subject for our components' subscriptions. It's an idea Rob is considering making "a thing" that seems to play nice with observables and lets you avoid storing the subscription in another variable.

What do you think about this @jelbourn